ros/src/twist_controller/dbw_node.py

```diff
@@ -113,6 +120,7 @@ def __init__(self):
         self.loop()
 
     def loop(self):
+        """Loop that computes throttle, brake and steer to publish."""
         rate = rospy.Rate(10)  # originally 50Hz
```

Comment:
do you think we shall go back to 50hz for carla?
dont remember for sure, but I think someone mentioned on slack this one better be 50hz.